php83-moodle-unit/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM alpine:edge
+
+ENV COMPOSER_HASH dac665fdc30fdd8ec78b38b9800061b4150413ff2e3b6f88543c636f7cd84f6db9189d43a81e5503cda447da73c7e5b6
+
+ENV PHPRUN_DEPS \
+  curl \
+  git \
+  libxml2-utils \
+  libxslt \
+  jq \
+  make \
+  mariadb-client \
+  openssh-client \
+  patch \
+  rsync \
+  sqlite \
+  wget
+
+RUN set -e \
+  && apk --update upgrade \
+  && echo 'https://dl-cdn.alpinelinux.org/alpine/edge/testing' >> /etc/apk/repositories \
+  && mkdir -p /var/www/html/web \
+  && addgroup -g 1000 -S web-group \
+  && adduser -u 1000 -D -S -G web-group web-user \
+  && mkdir -p /var/log/php82 && chown 1000:1000 /var/log/php82 \
```

Comment:
/var/log/php82 should be /var/log/php83 I guess

Reply:
Thank fixed, merging and pushing